elastica/dissipation.py

```diff
@@ -0,0 +1,117 @@
+__doc__ = """ Built in dissipation module implementations """
+__all__ = [
+    "DamperBase",
+    "ExponentialDamper",
+]
+
+from typing import Type, Union
+import numpy as np
+from abc import ABC, abstractmethod
+from elastica.rod import RodBase
+from elastica.rigidbody import RigidBodyBase
+
+
+class DamperBase(ABC):
+    """Base class for damping module implementations.
+
+    Notes
+    -----
+    All damper classes must inherit DamperBase class.
+
+
+        Attributes
+        ----------
+        system : RodBase or RigidBodyBase
+
+    """
+
+    _system: Union[Type[RodBase], Type[RigidBodyBase]]
```

Comment:
I think we should have separate typing alias in separate file. Could you maybe make `elastica/typing.py` and have `SystemType: Union[Type[RodBase], Type[RigidBodyBase]]`?